README.md

```diff
@@ -4,13 +4,29 @@
 
 2. `console.log` the length of a string.
 
-3. `console.log` the last character of any string. 
+3. `console.log` the last character of any string.
 
 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
 
 ```js
 'jimmy'
 // => 'jimmY'
+
+//Answers to 1-3 Lines 16-19
+let str = "pearl"
+console.log(str[0]);
+console.log(str.length);
+console.log(str.charAt(str.length-1));
```

Comment:
No need for charAt here. You can just do str[str.length - 1] and you will get the same result.